package.json

```diff
@@ -26,7 +27,8 @@
     ".prettierrc.js"
   ],
   "devDependencies": {
-    "eslint": "8.49.0"
+    "eslint": "8.49.0",
+    "@types/node": "20"
```

Comment:
in `.nvmrc` we specify this package to be on node 18. Can you either update that one too or pin this to `"18"`?

Reply:
Ah good catch, Ive updated it 👍 👍